pkg/prompts/prompts.go

```diff
@@ -500,6 +529,33 @@ func (r *realPrompter) CaptureAnyList(promptStr string, options any) (any, error
 	return r.CaptureList(promptStr, strArr)
 }
 
+func (*realPrompter) CaptureEmail(promptStr string, arg any) (any, error) {
+	prompt := promptui.Prompt{
+		Label:    promptStr,
+		Validate: validateEmail,
+	}
+
+	str, err := prompt.Run()
+	if err != nil {
+		return "", err
+	}
+
+	return str, nil
+}
+
+func (*realPrompter) CaptureEmpty(promptStr string, arg any) (any, error) {
```

Comment:
Not sure what `CaptureEmpty` means?

Reply:
You required that for these data, it is possible to just hit enter, which means it needs to be possible to allow empty values (empty strings). `CaptureString` does not allow empty strings.

Reply:
Why are you returning an `any` type. Shouldn't this be `CaptureStringAllowEmpty()` and return `string`?

Reply:
Yes that is probably a better suggestion. I used the `any` pattern because initially I had a scenario which was querying through `CaptureListDecision`.

We don't have that anymore so returning a string is way better now.